src/express/commands/start.js

```diff
@@ -100,6 +101,21 @@ async function installRequiredSoftwareOnRemoteMachines(
   await Promise.all(requirementTasks)
 }
 
+async function modifySshConfig(ip) {
+  console.log('📍Modifying ssh config in instance...')
+  const config = `TCPKeepAlive no
+  ClientAliveInterval 30
+  ClientAliveCountMax 240`
+  let command = `sudo sh -c 'cat << EOF >> /etc/ssh/sshd_config
+  ${config}'`
```

Comment:
can we avoid new line here?

Reply:
Guess adding in newline confuses the config for a filename : 
```
cat: TCPKeepAlive: No such file or directory
cat: no: No such file or directory
```